app/screens/global_threads/threads_list/index.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+import {AppStateStatus} from 'react-native';
+
+import {queryThreadsInTeam} from '@queries/servers/thread';
+import {observeTeammateNameDisplay} from '@queries/servers/user';
+
+import ThreadsList, {Tab} from './threads_list';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+type Props = {
+    tab: Tab;
+    teamId: string;
+    forceQueryAfterAppState: AppStateStatus;
```

Comment:
I think I like this, but would be great to have more context for my knowledge.

Reply:
This is for reflecting the DB changes in case the app goes to the background and there are changes made to the DB from the native side.